packages/adaptive_scaffold/pubspec.yaml

```diff
@@ -4,7 +4,6 @@ version: 0.0.1
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aopen+is%3Aissue+label%3A%22p%3A+adaptive_scaffold%22
```

Comment:
You will want to change this link to: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3Aadaptive_scaffold